src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1126,6 +1127,9 @@ private boolean promptUser(PromptBehavior prompt) {
         return prompt == PromptBehavior.Always || prompt == PromptBehavior.REFRESH_SESSION;
     }
     
+<<<<<<< HEAD
```

Comment:
How the code can compile if you have this in the code? There could be some conflicts with the your branch and the one you want to merge into. You need to resolve the conflicts.